tests/include/mocks/trafficmonitormock.hpp

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gmock/gmock.h>
+
+#include "aos/sm/networkmanager.hpp"
+
+using namespace aos::sm::networkmanager;
```

Comment:
consider moving defenition under namespace  instead of using

namespace aos::sm::networkmanager {
class MockTrafficMonitor : public TrafficMonitorItf {
}

Reply:
i did not see any difference in this UT, left as is.